### DISCUSSION TITLE
[Suggestion] add a toggle button when `autorefresh` is on?

### DISCUSSION THREAD

**Discussion by HaveF** [rclement/datasette-dashboards#145] [Ideas]
When I try to check the styles of the dashboard, I believe add a toggle button and the current time interval would be great.

![image](https://github.com/rclement/datasette-dashboards/assets/54462/9be88f7e-86e0-4d9e-a0d8-f75731a5b2dd)

I apologize for the poor quality of the image. It is only a representation, and I simply wanted to convey my intentions clearly.

**Comment by rclement** [comment]
I understand the frustration but FYI:
- The `autorefresh` dashboard option is optional and disabled by default.
- Enable it only when the dashboard is ready to go
- The initial use-case for it: dashboards end-users are displaying the dashboard in fullscreen on a monitor and data is changing at some regular and speedy pace.

I have already thought about this but I currently do not plan on making the option "dynamic" from the UI as it would require to maintain a parallel state compared to the static option from the metadata file.

**Comment by HaveF** [comment]
>
>    - Enable it only when the dashboard is ready to go
>
> I currently do not plan on making the option "dynamic" from the UI as it
> would require to maintain a parallel state compared to the static option
> from the metadata file
>

Thanks. It Make sense

> --
--
Sincerely,

HaveF
